### PR TITLE
Updated 

### PR DESCRIPTION
**Pull request by fasalmbt**
Updated 

- Reducing the risk to the team
- Official resources
-  Control rooms in Kerala
- State and district level organizations

To Hindi. 

Solves https://github.com/coronasafe/hindi/issues/2

### CODE REVIEW

File: contact/control-rooms-kerala.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
 ---
-description: >-
-  This page contains the contact information for the State-operated Control
-  Rooms in each district.
+विवरण: >-
```
Comment (ManasJayanth) [inline contact/control-rooms-kerala.md]:
```suggestion
description: >-
```
This is for the CMS system

File: contact/state-and-district-level-organizations.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
 ---
-description: >-
-  Gives the Contact Details of State Level Organizations handling the COVID-19
-  Crisis
+विवरण: >-
```
Comment (ManasJayanth) [inline contact/state-and-district-level-organizations.md]:
```suggestion
description: >-
```

File: contact/state-and-district-level-organizations.md [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
+  COVID-19 को संभालने वाले राज्य स्तरीय संगठनों के संपर्क विवरण को देखते हुए
+  संकट
 ---
 
-# State and District Level Organizations
+# राज्य और जिला स्तर के संगठन
 
-COVID-19 is kept in check by the timely and efficient intervention from the Local and State Government Organizations.
+सीओवीआईडी ​​-19 को स्थानीय और राज्य सरकार के संगठनों के समयबद्ध और कुशल हस्तक्षेप द्वारा जांच में रखा जाता है।
```
Comment (ManasJayanth) [inline contact/state-and-district-level-organizations.md]:
```suggestion
कोविद  ​​-19 को स्थानीय और राज्य सरकार के संगठनों के समयबद्ध और कुशल हस्तक्षेप द्वारा जांच में रखा जाता है।
```
I've seen Hindi newspapers use the acronym

File: resources/official-resources.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 ---
-description: >-
-  This page covers the Official Resources released by Governmental
-  Organisations.
+विवरण: >-
```
Comment (ManasJayanth) [inline resources/official-resources.md]:
```suggestion
description: >-
```

File: work/reducing-the-risk-to-the-team.md [hunk 5]
```diff
@@ -1,34 +1,34 @@
 ---
-description: How can you reduce the risk to team productivity in the midst of an outbreak
+विवरण: आप प्रकोप के बीच में टीम की उत्पादकता के जोखिम को कैसे कम कर सकते हैं?
```
Comment (ManasJayanth) [inline work/reducing-the-risk-to-the-team.md]:
```suggestion
description: आप प्रकोप के बीच में टीम की उत्पादकता के जोखिम को कैसे कम करने की सूज 
```

**Comment by ManasJayanth** [comment]
Thanks @fasalmbt Merging this soon. I have made some recommendations. Let me know if they are okay

**Comment by fasalmbt** [comment]
It's okay

**Pull request merged**